engine/class_modules/sc_death_knight.cpp

```diff
@@ -922,6 +923,7 @@ struct death_knight_t : public player_t {
     cooldown.empower_rune_weapon = get_cooldown( "empower_rune_weapon" );
     cooldown.icecap_icd          = get_cooldown( "icecap" );
     cooldown.pillar_of_frost     = get_cooldown( "pillar_of_frost" );
+    cooldown.raise_dead          = get_cooldown( "raise_dead" );
```

Comment:
You don't need to store a cooldown object for raise_dead if you're not going to adjust it throughout the simulation. One is automatically created on spell construction.